plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/meta/CubridMetaModel.java

```diff
@@ -53,100 +65,362 @@ public String getTableOrViewName(GenericTableBase base)
+        String description = JDBCUtils.safeGetString(dbResult, "comment");
+        Number lastValue = JDBCUtils.safeGetInteger(dbResult, "current_val");
+        Number minValue = JDBCUtils.safeGetInteger(dbResult, "min_val");
+        Number maxValue = JDBCUtils.safeGetInteger(dbResult, "max_val");
+        Number incrementBy = JDBCUtils.safeGetInteger(dbResult, "increment_val");
+        return new CubridSequence(container, name, description, lastValue, minValue, maxValue, incrementBy, dbResult);
+    }
+
+    @NotNull
+    @Override
+    public JDBCStatement prepareSynonymsLoadStatement(
+            @NotNull JDBCSession session,
+            @NotNull GenericStructContainer container)
+            throws SQLException {
+        String sql = "select * from db_synonym where synonym_owner_name = ?";
+        final JDBCPreparedStatement dbStat = session.prepareStatement(sql);
+        dbStat.setString(1, container.getName());
+        return dbStat;
+    }
+
+    @Nullable
+    @Override
+    public GenericSynonym createSynonymImpl(
+            @NotNull JDBCSession session,
+            @NotNull GenericStructContainer container,
+            @NotNull JDBCResultSet dbResult)
+            throws DBException {
+        String name = JDBCUtils.safeGetStringTrimmed(dbResult, "synonym_name");
+        String description = JDBCUtils.safeGetString(dbResult, "comment");
+        return new CubridSynonym(container, name, description, dbResult);
+    }
+
+    @NotNull
+    @Override
+    public boolean supportsDatabaseTriggers(@NotNull GenericDataSource dataSource) {
+        return true;
+    }
+
+    @NotNull
+    @Override
+    public boolean supportsTriggers(@NotNull GenericDataSource dataSource) {
+        return true;
+    }
+
+    @NotNull
+    @Override
+    public JDBCStatement prepareTableTriggersLoadStatement(
+            @NotNull JDBCSession session,
+            @NotNull GenericStructContainer container,
+            @Nullable GenericTableBase table)
+            throws SQLException {
+        String sql = "select t1.*, t2.*, owner.name from db_trigger t1 join db_trig t2 \n"
+                + "on t1.name = t2.trigger_name where owner.name = ? \n"
+                + (table != null ? "and target_class_name = ?" : "");
+        final JDBCPreparedStatement dbStat = session.prepareStatement(sql);
+        dbStat.setString(1, container.getName());
+        if (table != null) {
+            dbStat.setString(2, table.getName());
+        }
+        return dbStat;
+    }
+
+    @Nullable
+    @Override
+    public CubridTrigger createTableTriggerImpl(
+            @NotNull JDBCSession session,
+            @NotNull GenericStructContainer container,
+            @Nullable GenericTableBase table,
+            @Nullable String triggerName,
+            @NotNull JDBCResultSet dbResult)
+            throws DBException {
+        String name = JDBCUtils.safeGetString(dbResult, "name");

```

Comment:
Please create constants for "name" and "comment" in CubridConstants

Reply:
I have fixed this.